packages/components/stamp/src/stamp.spec.js

```diff
@@ -8,11 +8,22 @@ jest.mock('@commercetools-uikit/utils', () => ({
 }));
 
 it('should render the inline props', () => {
-  render(<Stamp tone="positive" label="Hello" icon={<AngleDownIcon />} />);
+  render(
+    <Stamp
+      tone="positive"
+      label="Hello"
+      icon={<AngleDownIcon data-testid="tested-icon" />}
+    />
+  );
   expect(screen.getByText('Hello')).toBeInTheDocument();
-  expect(screen.getByRole('img')).toBeInTheDocument();
```

Comment:
Do I have to be worried, could this happen in product-land as well?

Reply:
You mean how this would affect the MC ?

Reply:
from what I can tell, all the tests that use `screen.getByRole('img')` in the MC are testing for actual images, not icons